buildchain/buildchain/iso.py

```diff
@@ -115,6 +116,23 @@ def task__iso_add_node_manifest() -> types.TaskDict:
      }
 
 
+def task__iso_add_iso_manager() -> types.TaskDict:
+    """Copy the iso manager script to scripts."""
+    dest = constants.ISO_ROOT/'iso-manager.sh'
+    iso_manager = [
+            constants.ROOT/'scripts'/'iso-manager.sh',
+            dest
+            ]
+    return {
+            'title': lambda task: utils.title_with_target1('GENERATE', task),
+            'actions': [(coreutils.cp_file, iso_manager)],
+            'targets': [dest],
+            'task_dep': ['_iso_mkdir_root'],
+            'uptodate': [True],
```

Comment:
Nope.
Why do you want it to be always true?

If the script is modified, you want to copy it again.
You should have `constants.ROOT/'scripts'/'iso-manager.sh'` as `file_dep`.

Reply:
And looking at the code, it seems like `task__iso_add_node_manifest` was wrong as well and should be fixed (in another commit/PR, not here).